### PR TITLE
[INLONG-8997][Sort] Keep the logic the same of mysql with flink v1.13 in end to end test

### LINKED ISSUE #8997: [Improve][Sort] Keep the logic the same with flink v1.15 mysql in flink v1.13 end to end test
### Description

Keep the flink v1.13 test logic of MysqlContainer SQLServer Container the same with flink v1.15.
Linked issue: #8967 

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liaosunny123**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #8997 

### Motivation

The mysql test container existed before this pr and there are some unneed codes, so I remove them.

As for MSSQL test container, it can help others write sqlserver test container for sqlserver Connector in flink 1.13. Just like the Mysql test container in flink 1.13 end to end test which are not used.

These container are tested in flink 1.15 end to end test.

### CODE REVIEW

**Comment by EMsnap** [comment]
Please add a description on what problem the pr solves, btw why add a class that's not used ?

**Comment by liaosunny123** [comment]
The mysql test container existed before this pr and there is some u need code, so I remove them.

As for MSSQL test container, it can help others write sqlserver test container for sqlserver Connector in flink 1.13. Just like the Mysql test container in flink 1.13 but not used.

These container are tested in flink 1.15 end to end test.

**Comment by liaosunny123** [comment]
> Please add a description on what problem the pr solves, btw why add a class that's not used ? 

The mysql test container existed before this pr and there is some u need code, so I remove them.

As for MSSQL test container, it can help others write sqlserver test container for sqlserver Connector in flink 1.13. Just like the Mysql test container in flink 1.13 but not used.

These container are tested in flink 1.15 end to end test.

**Comment by EMsnap** [comment]
> > Please add a description on what problem the pr solves, btw why add a class that's not used ?
> 
> The mysql test container existed before this pr and there are some unneed codes, so I remove them.
> 
> As for MSSQL test container, it can help others write sqlserver test container for sqlserver Connector in flink 1.13. Just like the Mysql test container in flink 1.13 end to end test which are not used.
> 
> These container are tested in flink 1.15 end to end test.

Thanks for the contribution, but we only submit code that's needed for a feature or so. Maybe you could submit this class when implement sqlserver for flink1.13

**Comment by liaosunny123** [comment]
> > > Please add a description on what problem the pr solves, btw why add a class that's not used ?
> > 
> > The mysql test container existed before this pr and there are some unneed codes, so I remove them.
> > 
> > As for MSSQL test container, it can help others write sqlserver test container for sqlserver Connector in flink 1.13. Just like the Mysql test container in flink 1.13 end to end test which are not used.
> > 
> > These container are tested in flink 1.15 end to end test.
> 
> Thanks for the contribution, but we only submit code that's needed for a feature or so. Maybe you could submit this class when implementing sqlserver for flink1.13 (because U need to UT this in flink 1.13) 

sqlserver is implemented in flink 1.13, it lacks UT. Do you mean when adding sqlserver Connector of flink 1.13 UT?

**Comment by EMsnap** [comment]
> > > > Please add a description on what problem the pr solves, btw why add a class that's not used ?
> > > 
> > > 
> > > The mysql test container existed before this pr and there are some unneed codes, so I remove them.
> > > As for MSSQL test container, it can help others write sqlserver test container for sqlserver Connector in flink 1.13. Just like the Mysql test container in flink 1.13 end to end test which are not used.
> > > These container are tested in flink 1.15 end to end test.
> > 
> > 
> > Thanks for the contribution, but we only submit code that's needed for a feature or so. Maybe you could submit this class when implementing sqlserver for flink1.13 (because U need to UT this in flink 1.13)
> 
> sqlserver is implemented in flink 1.13, it lacks UT. Do you mean when adding sqlserver Connector of flink 1.13 UT?

I mean MSSQL test container can save for later pr about MSSQL, it's not used, if submitted, you should also add ut to this class on flink 1.13

File: inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/utils/MySqlContainer.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public MySqlContainer() {
 
     public MySqlContainer(MySqlVersion version) {
         super(DockerImageName.parse(IMAGE + ":" + version.getVersion()));
-        addExposedPort(MYSQL_PORT);
+        addFixedExposedPort(33306, 3306);
```
Comment (EMsnap) [inline inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/utils/MySqlContainer.java:52]: what is 33306 for ?

File: inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/utils/MySqlContainer.java [hunk 2]
```diff
@@ -60,8 +60,6 @@ protected Set<Integer> getLivenessCheckPorts() {
     @Override
     protected void configure() {
         // HERE is the difference, copy to /etc/mysql/, if copy to /etc/mysql/conf.d will be wrong
```
Comment (EMsnap) [inline inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.13/src/test/java/org/apache/inlong/sort/tests/utils/MySqlContainer.java:62]: remove this comment

**Pull request closed without merging**